package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "node-neutralino",
+  "package-name": "node-neutralino",
+  "version": "1.1.3",
+  "description": "wrapper project for neutralinojs backend app",
+  "entry point": "./src/index.js",
+  "main": "./src/index.js",
+  "license": "MIT",
+  "author": {
+    "name": "Neutralinojs and contributors",
```

Comment:
I think you can use "<Your name>, Neutralinojs contributors" since this is a community project

Reply:
Done 😊